apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/ast/AST.kt

```diff
@@ -0,0 +1,121 @@
+package com.apollographql.apollo.compiler.ast
+
+data class AST(
+  val enums: List<EnumType>,
+  val customTypes: Map<String, String>,
+  val inputTypes: List<InputType>,
+  val fragments: List<FragmentType>,
+  val operations: List<OperationType>
+) {
+  data class OperationType(
+    val name: String,
+    val type: Type,
+    val definition: String,
```

Comment:
do you think we can gain readability from using type alias for some of these scalar fields?